chart/templates/jobserviceaccount.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if ((.Values.jobConfig).serviceAccount).create | default true }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "job-executor-service.jobConfig.serviceAccountName" . }}
+  labels:
+    {{- include "job-executor-service.labels" . | nindent 4 }}
+  {{- with ((.Values.jobConfig).serviceAccount).annotations }}
+  annotations:
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+---
+# Role for the jobs
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: job-executor-service_job
+  namespace: {{ .Release.Namespace }}
+rules: []
```

Comment:
I am wondering if we want to give the option to customize the set of permissions when performing `helm install/upgrade` similar to what we want to do with the security context :thinking: 
It would basically mean having the array of rules in values.yaml (empty by default but users can customize it when installing the JES)
Here the feedback of @christian-kreuzberger-dtx  would be very appreciated :smile:

Reply:
If we let the user configure the service accounts, I would prefer to have two different service accounts. One with an empty set of rules for all jobs that don't need access to the Kubernetes APIs and another one that can be configured when installing JES. The additional service account can be used for jobs if `jobConfig.enableKubernetesApiAccess=true` is used instead of using the JES service account.

Reply:
For now the empty rules array is perfectly fine as the default setting. The goal is to restrict default access of workloads. 
There is another story where we will allow users to overwrite the service account for specific workloads.